sympy/core/numbers.py

```diff
@@ -1987,7 +1987,7 @@ def __new__(cls, expr, coeffs=Tuple(), alias=None, **args):
 
         dom = minpoly.get_domain()
 
-        if coeffs != Tuple():
+        if coeffs != None:  # Empty list or tuple is accepted
```

Comment:
Would that form be more efficient? I noticed that ``if coeffs:`` is not correct/